ui/app/models/pki/tidy.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+
+@withFormFields()
```

Comment:
We typically use this decorator when dynamically rendering the form inputs for a model. It looks like they're explicitly written out for each attribute, so I think we can remove this decorator for now?